org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/sso/filter/SSOWebuiFilter.java

```diff
@@ -0,0 +1,133 @@
+/******************************************************************************
+ * Copyright (C) 2016 Logilite Technologies LLP								  *
+ * This program is free software; you can redistribute it and/or modify it    *
+ * under the terms version 2 of the GNU General Public License as published   *
+ * by the Free Software Foundation. This program is distributed in the hope   *
+ * that it will be useful, but WITHOUT ANY WARRANTY; without even the implied *
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.           *
+ * See the GNU General Public License for more details.                       *
+ * You should have received a copy of the GNU General Public License along    *
+ * with this program; if not, write to the Free Software Foundation, Inc.,    *
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.                     *
+ *****************************************************************************/
+package org.adempiere.webui.sso.filter;
+
+import java.io.IOException;
+import java.util.logging.Level;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.adempiere.base.sso.ISSOPrinciple;
+import org.adempiere.base.sso.SSOUtils;
+import org.compiere.model.MSysConfig;
+import org.compiere.util.CLogger;
+
+/**
+ * Request filter class for the SSO authentication
+ * 
+ * @author Logilite Technologies
+ */
+public class SSOWebuiFilter implements Filter
```

Comment:
Following existing naming convention, this should be named as `SSOWebUIFilter` instead.

Reply:
Done